### DISCUSSION TITLE
RTMV, symbolic link creator/move script?

### DISCUSSION THREAD

**Discussion by ghost** [kannibalox/pyrosimple#35] [General]
Need to spend more time browsing, but very glad to have bumped into this project. Was curious if the classic command "rtmv" was implemented in pyrosimple? Tried doing a 4 character search of the source, but github didnt yield and results.

**Comment by kannibalox** [comment]
No, I removed it during the "slimming down" phase of this project because it was really just a fancy way of creating symlinks, and didn't really have much connection to however rtorrent was running. I've been mulling over ways to try and get it integrated back into rtcontrol (maybe with a `--move`/`--move-type` flag?), but I may just end up reintroducing it in a different form.

**Reply by ghost** [reply]
Okay, good to know, thanks for explanation and fully understand the slimming down. Personally use rtmv heavily in my workflow so I'll be looking forward to seeing how this project evolves. Much appreciated.

**Reply by kannibalox** [reply]
I haven't tested it extensively, but if you want to try a simple conversion of the script, it's available in the most recent code:
```
pip install git+https://github.com/kannibalox/pyrosimple.git
python -m pyrosimple.scripts.rtmv_legacy --help
```
